tools/travis/test_openwhisk.sh

```diff
@@ -86,6 +86,7 @@ cd $OPENWHISK_HOME
 #./tools/travis/setup.sh
 
 #  Fire up the cluster
+echo 'limit_invocations_per_minute: 120' >> $OPENWHISK_HOME/ansible/environments/local/group_vars/all
```

Comment:
Due to travis VM network is not stable, so added this configuration make below relative test case stable
```
org.apache.openwhisk.core.cli.test.ApiGwCliTests > Cli Wsk api creation with path parameters with swagger should verify delete basepath/path FAILED
    org.scalatest.exceptions.TestFailedException: /home/travis/gopath/src/github.com/apache/openwhisk/bin/wsk -i --apihost 172.17.0.1 --apiversion v1 api create --auth 23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP /CLI_APIGWTEST11_bp /path get CLI_APIGWTEST11_action --apiname CLI_APIGWTEST11 API Name
    error: Unable to create API: Too many requests in the last minute (count: 64, allowed: 60). (code aa221bc7c98c15b86614fd28bae0c013)
    exit code: 1 was not equal to 0
        at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:343)
        at org.scalatest.Matchers$AnyShouldWrapper.shouldBe(Matchers.scal
```